ThirdPartyHosting.md

```diff
@@ -8,6 +8,7 @@ In layman words, think of this feature as a payment processor factory which allo
 
 ## The list of BTCPay third-party hosts
 
+* [btcpaynet.com](https://btcpaynet.com)
```

Comment:
The list is in alphabetical order. So your entry needs to be after btcpayjungle.

Reply:
Done. Thank you :-)